src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -64,6 +64,7 @@ public DocumentDBAttribute(string databaseName, string collectionName)
         /// Optional. A string value indicating the app setting to use as the DocumentDB connection string, if different
         /// than the one specified in the <see cref="DocumentDBConfiguration"/>.
         /// </summary>
+        [AutoResolve(AllowTokens = false)]
         public string ConnectionStringSetting { get; set; }
```

Comment:
It is somewhat strange for such AutoResolve properties that the property name is ConnectionStringSetting, but the value after resolution is a connection string. I.e. you're usint resolvedAttr.ConnectionStringSetting as a connection string. Oh well.